src/commands/slack-circleci-build.yml

```diff
@@ -19,12 +19,16 @@ parameters:
     type: string
     default: ''
     description: Textual reference for a prefixed emoji to be formatted into the message to Slack.
+  link:
+    type: string
+    default: "https://app.circleci.com/pipelines/workflows/$CIRCLE_WORKFLOW_ID"
```

Comment:
Does the $CIRCLE_WORKFLOW_ID value get populated when using the default? I vaguely remember having trouble with this before but my brain is still booting up for the new year.

Reply:
It does work! (I tested that scenario out just now to be extra sure - I may have tested it before the break, but holiday brain yes...)

looks like everything involved evaluates the env var properly (vs some clever shell scripter thinking they were awesome and using single quotes somewhere)